fe/fe-core/src/main/java/com/starrocks/alter/SchemaChangeHandler.java

```diff
@@ -2305,6 +2306,10 @@ private void processAddIndex(CreateIndexClause alterClause, OlapTable olapTable,
             return;
         }
 
+        if (newIndex.getIndexType() == IndexType.GIN && olapTable.enableReplicatedStorage()) {
+            throw new SemanticException("GIN is only supported in not replicated mode");
```

Comment:
`GIN does not support replicated mode` or `replicated mode does not support GIN`